extensions/2.0/Vendor/FB_materials_lightmap/README.md

```diff
@@ -0,0 +1,80 @@
+# FB\_materials\_lightmap
```

Comment:
Consider renaming. Additional ideas, aside from those mentioned in the thread:

- FB_materials_base_color_multiplier
- FB_materials_unlit_lightmap

Reply:
Thank you for the suggestions!